js/script.js

```diff
@@ -300,26 +300,32 @@ function landItem(time) {
     }
 }
 
+/**
+ * Generates random numbers to be utilized for array indices
+ * @param {*} arr the array to be randomly accessed
+ * @param {*} compareItem the current item to ensure no two items are outputted consecutively
+ * @param {*} beforeStartIndex one less than the start index of the subarray to access
+ * @returns a random number that serves as the index to randomly access an array
+ */
+function randomNumberGenerator(arr, compareItem, beforeStartIndex) {
```

Comment:
Might be good to just have a quick sanity check that arr is an array, compareItem is a string or whatever it is, and beforeStartIndex is an int